.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

Comment:
this seems a little aggressive. We could run these twice a day at most I think?

`0 0,12 * * *`

Do you think this needs to be checking more frequently for any particular reason?